setup/cocci.m4

```diff
@@ -285,7 +285,7 @@ AC_DEFUN([AC_COCCI_TOOL],
     AC_SUBST([$1], [no])
   ],
   [dnl  find the tool
-    AC_COCCI_FINDTOOL([$1],[[$]$1])
+    AC_COCCI_FINDTOOL([$1],[$2])
```

Comment:
How do you think about to improve this script more than a single macro call (around the specification "`[$]$1`")?